app/views/trips/_form.html.erb

```diff
@@ -1,13 +1,45 @@
-<div class="container">
+<table class="form-table">
```

Comment:
And again ^

Reply:
I changed all the tables to be div with lists and I adapted the scss. All in sha: 3361eb6